backend/src/main/java/cz/muni/fi/pa165/pneuservis/backend/entity/Customer.java

```diff
@@ -1,7 +1,167 @@
 package cz.muni.fi.pa165.pneuservis.backend.entity;
 
-import javax.persistence.Entity;
+import cz.muni.fi.pa165.pneuservis.backend.enums.VehicleTypeEnum;
 
-//@Entity
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Set;
+
+/**
+ * @author Zuzana Žufanová, zufinka@mail.muni.cz
+ */
+
+@Entity
 public class Customer {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotNull
+    private String email;
+
+    @NotNull
+    private String name;
+
+    @NotNull
+    private String surname;
+
+    @NotNull
+    private String city;
+
+    @NotNull
+    private String street;
+
+    @NotNull
+    @Size (min = 5, max = 5)
+    private Integer psc;

```

Comment:
There really shouldn't be any other language than English, so please change this to "zipCode". Also, I think it'd be a lot better to have it as an String instead. My reasoning is that it helps us with a lot of possible edge cases (first digit being 0 e.g.) and we don't really lose anything by not having it as an Integer since we're not going to perform any arithmetic operations on it anyway.